### PR TITLE
Add ARRCON to Games

### PR DESCRIPTION
**Pull request by radj307**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software)
  must be added to `non-free.md` and marked `⊘ Proprietary`:
  ``- [Name](http://homepage/) `⊘ Proprietary` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Comment by nodiscc** [comment]
It's a client, not server software - I'm not sure we should list those. By the same reasoning `RconCli` should probably not be listed.

**Comment by radj307** [comment]
I agree about clients in general not really belonging on a list of selfhosted server software.

However, I would argue that server administration utilities related to communication do still deserve a place on this list (albeit maybe in a subcategory), since they are still related to self-hosted game software & data transparency.

**Comment by nodiscc** [comment]
> server administration utilities related to communication do still deserve a place on this list

I think they do. Please create a new section `Game server control panels` (o find a better name), and move ARRCON and RconCli there. The guidelines say:


These projects were also mentioned:
- https://linuxgsm.com/
- https://pterodactyl.io





This would fix https://github.com/awesome-selfhosted/awesome-selfhosted/issues/1555 and https://github.com/awesome-selfhosted/awesome-selfhosted/issues/2152

**Comment by nodiscc** [comment]
Pterodatcyl is being added in https://github.com/awesome-selfhosted/awesome-selfhosted/pull/2945/files, you can simply copy the added line from there (and add it to the new section), we'll remove it from https://github.com/awesome-selfhosted/awesome-selfhosted/pull/2945

**Comment by radj307** [comment]
I created a new section that should be broad enough to accommodate all of those - I went with 'Games - Administrative Utilities & Control Panels' to emulate the other categories that follow a similar pattern, like Communication.

**Comment by kokomo123** [comment]
I believe LinuxGSM has already been mentioned in there, https://github.com/d1nuc0m/awesome-selfhosted/commit/5524367f64cf043dbb6dc3e89d48cd3bed528aa4 is the direct commit. I'll create a separate PR for Minetest 👍🏻

**Comment by radj307** [comment]
Thanks - I added LinuxGSM to the category as well.

File: README.md [hunk 1]
```diff
@@ -961,13 +959,22 @@ Games, game servers and control panels.
...
 - [Zero-K](https://zero-k.info/) - Open Source on Springrts engine. Zero-K is a traditional real time strategy game with a focus on player creativity through terrain manipulation, physics, and a large roster of unique units - all while being balanced to support competitive play. ([Source Code](https://github.com/ZeroK-RTS/Zero-K)) `GPL-2.0` `Lua`
 
 
+### Games - Administrative Utilities & Control Panels
+
+**[`^        back to top        ^`](#)**
+
+- [ARRCON](https://github.com/radj307/ARRCON) - Terminal-based RCON client compatible with any game servers using the Source RCON Protocol. `GPL-3.0` `C++`
+- [LinuxGSM](https://linuxgsm.com/) - CLI tool for deployment and management of dedicated game servers on Linux: more than 120 games are supported. ([Source Code](https://github.com/GameServerManagers/LinuxGSM)) `MIT` `Shell`
+- [Pterodactyl](https://pterodactyl.io/) - Management panel for game servers, with an intuitive UI for end users. ([Source Code](https://github.com/pterodactyl/panel)) `MIT` `PHP`
```
Comment (nodiscc) [inline README.md:974]: I just merged https://github.com/awesome-selfhosted/awesome-selfhosted/pull/2945 which adds Pterodactyl in another section. This should be rebased on the latest `master` and the duplicate Pterodactyl entry should be removed.

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Comment by radj307** [comment]
rebased on latest `master` & removed Pterodactyl

**Pull request merged**